schema/src/main/scala/org/constellation/schema/snapshot/HeightRange.scala

```diff
@@ -0,0 +1,15 @@
+package org.constellation.schema.snapshot
+
+import io.circe.{Decoder, Encoder}
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+
+/**
+  * Inclusive range of heights
+  */
+case class HeightRange(from: Long, to: Long)
+
+object HeightRange {
+  val maxRange = HeightRange(0, Long.MaxValue)
```

Comment:
Maybe `val MaxValue = HeightRange(0, Long.MaxValue)` to be consistent with naming (`Long.MaxValue`, `HeightRange.MaxValue`)

Reply:
I was thinking the same. I will change it to `MaxValue`